netdisco/ssdp.py

```diff
@@ -253,6 +253,8 @@ def scan(timeout=DISCOVER_TIMEOUT):
             for sock in ready:
                 try:
                     response = sock.recv(1024).decode("utf-8")
+                except UnicodeDecodeError:
+                    continue
```

Comment:
Instead of just continue, can you also add this

```python
logging.getLogger(__name__).debug(
    'Ignoring invalid unicode response from %s', sock.getpeername())
```

Reply:
These are UDP sockets so I changed it to this, assuming you want the host that sent the bad data:

                try:
                    data, address = sock.recvfrom(1024)
                    response = data.decode("utf-8")
                except UnicodeDecodeError:
                    logging.getLogger(__name__).debug(
                        'Ignoring invalid unicode response from %s', address)
                    continue

Reply:
Yeah, that would be perfect.